pipeline-modules/app-service/aws/semgrep.yaml

```diff
@@ -0,0 +1,67 @@
+provisioner: aws
+name: semgrep
+version: 1
+revision: 1
+displayName: semgrep
+description: Run semgrep scans
+target: ""
+category: scan
+keywords:
+  - scan
+  - sast
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute the bash script from
+      type: string
+      default: repo
+    allow_critical:
+      title: Allow Critical Vulnerabilities
+      description: Allows builds with critical vulnerabilites
+      type: boolean
+      default: false
+    allow_error:
+      title: Allow Error Vulnerabilities
+      description: Allows Error vulnerabilites
+      type: boolean
+      default: false
+template: |
+  version: 0.2
+  env:
+    shell: bash
+  phases:
+    build:
+      commands:
+        - python3 --version
```

Comment:
add `cd $CODEBUILD_SRC_DIR/{{ working_dir }}` in the start of each module otherwise, the module will only work for apps that have code on repo root OR one of the previous module did the cd.

Reply:
Weren't we going to change it in the git clone script itself? Won't that be better?